core/controllers/pages.py

```diff
@@ -40,6 +42,11 @@
     'site_forum_url', {'type': 'unicode'},
     'The site forum URL (for links; the Forum page is configured separately)',
     default_value='https://site/forum/url')
+MODERATOR_REQUEST_FORUM_URL = config_domain.ConfigProperty(
+    'moderator_request_forum_url', {'type': 'unicode'},
+    'A link to the forum for nominating explorations to be featured '
```

Comment:
I don't think that's needed. We have other similar config properties that don't come with such comments, and I don't really want to risk that comment getting out of date (especially since we may well refer to it from somewhere else in the future).